### DISCUSSION TITLE
Remove whitespace from start and end of environment variables

### DISCUSSION THREAD

**Discussion by JorgenSQ** [vercel/community#401] [Idea]
To save some penguins, and some developers sanity, please use .trim() or the equivalent to remove whitespace from environment variables at the start end end of strings.

**Comment by swarnava** [comment]
Hi @JorgenSQ,

Thank you for bringing this up. I just checked internally and can confirm that our team is aware of this issue and it is added as feature request in our internal tracker. Unfortunately, I don't have any concrete ETA when we will implement this.

Have a good day!
